### ISSUE TITLE
[a11y-defect-3]: Your VA Payments breadcrumbs should add a current link instead of replacing the previous link

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#95163]
### Point of contact

Trevor Pierce

### Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

### Details

The breadcrumb in the [Your VA Payments](va-payment-history/payments/) view updates the link title and swaps out the previous current link instead of adding a third link. This was confusing because the breadcrumbs did not behave consistently when I tried to go back to the previous page. This could have an impact on screen reader, keyboard, and voice command users. Screen shots attached below.

---

![Image](https://github.com/user-attachments/assets/b416d658-029e-4397-8ef0-a721d8f35385)

---

![Image](https://github.com/user-attachments/assets/f242a404-456e-4933-a4e8-5f6e2896fce1)

### Reproduction steps

1. Log into a test user account and navigate to the `va-payment-history/` view 
2. Click the "View your VA payments history" link
3. Observe the breadcrumb text updates but does not add a third breadcrumb
4. Also observe clicking the breadcrumb does not go backward to the previous page

### Proposed solution or next steps

Ideally the breadcrumb list adds a third breadcrumb when users click "View your VA payments history" link. It would also be helpful if this page to view payments had a more descriptive heading that was dissimilar from the previous page, so the breadcrumb links could also be more distinct.

### References, articles, or WCAG support

1. Understanding [SC 2.4.4](https://w3.org/TR/WCAG22#link-purpose-in-context): Link Purpose (In Context) (Level A)

### Type of issue

- [ ] Axe-core
- [ ] Screenreader | assistive tech & device support
- [ ] Keyboard
- [ ] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Focus
- [ ] Headings
- [ ] Content style guide
- [x] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [x] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else

**Comment by dysbo** [comment]
Refinement notes:
- Need to discuss with content folks (@jstrothman, @rallencf, others?) about making the text change outlined above
- Breadcrumbs appear to live in the `content-build` repository; see if we can find documentation on how those are constructed

**Comment by dysbo** [comment]
@rallencf to meet with CAIA to discuss and determine next steps.

**Comment by rallencf** [comment]
@1Copenut @dysbo @jstrothman I found another breadcrumb issue. From the "Download VA benefit letters page" (in staging) if you click on "Get your VA benefits" the breadcrumb properly adds the next page to the breadcrumb trail, but the text in link 2 of the breadcrumb trail changes from "Records" to "VA records" and then links to a different records page not found in the original navigation path.

![Image](https://github.com/user-attachments/assets/0205d11b-276f-485e-8ce4-e2466c29b559)
![Image](https://github.com/user-attachments/assets/189dea1a-b09a-41fa-9654-a159714eb8d3)

**Comment by rallencf** [comment]
Requested a brief chat with CAIA in slack to get their input. I also wonder if breadcrumb issues are a broader issue than these couple of pages.

**Comment by jstrothman** [comment]
> [@1Copenut](https://github.com/1Copenut) [@dysbo](https://github.com/dysbo) [@jstrothman](https://github.com/jstrothman) I found another breadcrumb issue. From the "Download VA benefit letters page" (in staging) if you click on "Get your VA benefits" the breadcrumb properly adds the next page to the breadcrumb trail, but the text in link 2 of the breadcrumb trail changes from "Records" to "VA records" and then links to a different records page not found in the original navigation path.
> 
> ![Image](https://github.com/user-attachments/assets/0205d11b-276f-485e-8ce4-e2466c29b559) ![Image](https://github.com/user-attachments/assets/189dea1a-b09a-41fa-9654-a159714eb8d3)

@rallencf Great that you found this--I think it could use a separate GH issue

**Comment by dysbo** [comment]
Thanks @rallencf for finding that other breadcrumb issue - let's create a separate GH issue for that one.  I believe @1Copenut is looking at that page in https://github.com/department-of-veterans-affairs/va.gov-team/issues/95086 (correct me if I'm wrong) so he may have already started one.

**Comment by rallencf** [comment]
I scheduled a chat with CAIA for Thursday to discuss both breadcrumb issues.

**Comment by rallencf** [comment]
Met with CAIA and @1Copenut to review breadcrumb issues. We're all in agreement about the desired content and functionality. CAIA is close to publishing breadcrumb guidance in the design system that we'll reference for updating the issues in this ticket.

**Comment by dysbo** [comment]
We will wait for the guidance on breadcrumbs to be finalized and published before we action on this to avoid rework.

**Comment by salihaghaffar** [comment]
@rallencf @1Copenut @dysbo tagging you guys to put a reminder for a follow up on the above.

**Comment by dysbo** [comment]
Thanks to @rallencf and @1Copenut for helping me find these links 😄 

Breadcrumb guidance has been updated, seen on the VA Design System site [here](https://design.va.gov/components/breadcrumbs).

Per the guidance, we should look into updating the breadcrumbs on the **Your VA Payments** page to display
```
VA.gov home > View your VA payment history > Your VA payments
```

Additional discussion in this ticket involve potentially updating the title of the **Your VA payments** page to differentiate it better from the previous page.  I would like @rallencf's input on that piece of it before we move on changing any of the page titles.

**Also note**: it is likely that we will have to make these changes outside of the `vets-website` repo and may need to familiarize with another build/approval process for breadcrumb updates, as I believe they are part of the site-wide static content build rather than the React code we typically work in.

**Comment by rallencf** [comment]
@janechodance @sh0wn Correct breadcrumb:

Page:
https://staging.va.gov/va-payment-history/payments/

Breadcrumb:
**- Current:** VA.gov homeView VA payment history
**- Change to:** VA.gov home > View VA payment history > Your VA payments

**Comment by dysbo** [comment]
Verified in staging that the breadcrumbs are correct and functional

<img width="1600" height="962" alt="Image" src="https://github.com/user-attachments/assets/f28c8bbb-221b-415c-ac1f-12dcce7cf8f3" />
